apps/palette-generator/palette-generator.html

```diff
@@ -0,0 +1,72 @@
+<html>
+<head>
+  <title>Palette Generator</title>
+  <style>
+    /*
+     * Needed to override palette button styles.
+     * TODO: modify palette's index.scss to add a class specifically for palette
+     * buttons, something like `button.palette { ... }`.`
+     */
+    button.usual {
+      text-transform: none;
+      overflow: visible;
+      font-family: serif;
+      font-size: inherit;
+      margin: 0;
+      padding-top: 0.25em;
+      padding-left: 0.25em;
+      padding-right: 0.25em;
+      padding-bottom: 0.25em;
+      color: #000;
+    }
+    button.usual:disabled {
+      pointer-events: none;
+    }
+    button.usual:hover {
+      background-color: darkGray;
+    }
+  </style>
+</head>
+<body style="background-color: white; margin-left: 1em;">
+  <script type="module">
+    import "./palette-generator.ts";
+  </script>
+  <h1>Palette Generator</h1>
+  <div style="width:50%">
+    <fieldset>
+      <legend>Settings</legend>
+      <p>
+        <label for="paletteName">Palette name:</label><br>
+        <input type="text" id="paletteName" value="palette">
+      </p>
+      <p>
+        <label for="rowStart">Starting row (>=1):</label><br>
+        <input type="number" id="rowStart" min="1" max="100" value="1">
+      </p>
+      <p>
+        <label for="colStart">Starting column (>=1):</label><br>
+        <input type="number" id="colStart" min="1" max="100" value="1">
+    </fieldset>
+    <p>
+      List the items in each row, one line per row.  Each row consists of a set
+      of glosses separated by a space.
+    </p>
+      <label for="glossInput">Glosses:</label><br>
```

Comment:
The label for the text area to receive BCI-AV-ID, glosses, SVGs etc should no longer be "Glosses" since it accepts a lot more than glosses.

Reply:
Right, I'll come up with a better name.